src/NovelLoggingService.h

```diff
@@ -0,0 +1,62 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT License (MIT). See LICENCE.md in the repository root for more information.
+
+#ifndef NOVELRT_NOVELLOGGINGSERVICE_H
+#define NOVELRT_NOVELLOGGINGSERVICE_H
+
+#include <iostream>
+#include "spdlog/spdlog.h"
+#include "spdlog/sinks/stdout_color_sinks.h"
+
+namespace NovelRT {
+  enum LogLevel {
+    DEBUG = SPDLOG_LEVEL_DEBUG,
+    INFO = SPDLOG_LEVEL_INFO,
+    WARN = SPDLOG_LEVEL_WARN,
+    ERR = SPDLOG_LEVEL_ERROR,
+    OFF = SPDLOG_LEVEL_OFF
```

Comment:
Please rename the states in `PascalCase` as per our convention instead of `SCREAMING_IM_ON_FIRE_CASE` so `ERR` becomes `Err` or `Error`, `DEBUG` becomes `Debug` and so on.